end_users/README.md

```diff
@@ -1,3 +1,24 @@
 # End Users
 
-Documentation coming soon...
+## Table of Contents
+
+* [Merge Options](#merge-options)
+  * [Setting the Correct Merge Option](#setting-the-correct-merge-option)
+
+## Merge Options
+
+Repositories that use BonnyCI should not use any merge strategy which changes the SHA of the commit being merged in. The reasoning for this is that [Zuul](https://github.com/openstack-infra/zuul), the scheduler used in BonnyCI, will be thrown off by the change in SHA when it merges commits that successfully pass the gate queue.
+
+GitHub presents repository owners with 3 choices for what happens when a commit is merged from a pull request:
```

Comment:
GitHub presents repository owners with 3 choices for what happens when merging a pull request: